runtime/vm/MHInterpreter.cpp

```diff
@@ -1353,6 +1366,127 @@ VM_MHInterpreter::getArgSlotsBeforePosition(j9object_t argumentTypes, U_32 argPo
 	return argumentSlots;
 }
 
+j9object_t
+VM_MHInterpreter::filterArgumentsWithCombiner(j9object_t methodHandle)
+{
+	/* Get filterHandle.type and filterHandle.type.argSlots */
+	/* [... filterHandle args] */
+	j9object_t filterHandleType = getMethodHandleMethodType(methodHandle);
+	j9object_t argumentTypes = getMethodTypeArguments(filterHandleType);
+	U_32 filterArgSlots = getMethodTypeArgSlots(filterHandleType);
+	j9object_t argumentIndices = J9VMJAVALANGINVOKEFILTERARGUMENTSWITHCOMBINERHANDLE_ARGUMENTINDICES(_currentThread, methodHandle);
+	U_32 argumentIndicesCount = J9INDEXABLEOBJECT_SIZE(_currentThread, argumentIndices);
+	UDATA *spFirstFilterArgSlot = _currentThread->sp + filterArgSlots;
+
+	/* [... filterHandle args descriptionBytes MethodTypeFrame] */
+	UDATA *spPriorToFrameBuild = _currentThread->sp;
+	(void)buildMethodTypeFrame(_currentThread, filterHandleType);
+
+	/* [... filterHandle args descriptionBytes MethodTypeFrame filterHandle args] */
+	_currentThread->sp -= (filterArgSlots  + 1);
+	memcpy(_currentThread->sp, spPriorToFrameBuild, sizeof(UDATA) * (filterArgSlots+1));
```

Comment:
I don't think this memcpy of the arguments is actually used.  There's also nothing that describes them on the stack so if they were used, we'd have a potential issue post-gc.

Can you try removing this?